### ISSUE TITLE
May I add Karatsuba? 

### ISSUE THREAD

**Issue by eashsaxena** [pedrovgs/Algorithms#58]
It would be helpful for you to add Karatsuba as it is used to get fastest multiplication.

**Comment by pedrovgs** [comment]
Indeed! I've seen your PR but you need to code and not add just a file with some information and code inside. Please, take a look at my comment in the PR 😃
